scripted/plugin/src/main/scala/sbt/ScriptedPlugin.scala

```diff
@@ -68,15 +75,18 @@ object ScriptedPlugin extends AutoPlugin {
       ModuleUtilities.getObject("sbt.test.ScriptedTests", loader)
     }
 
-  def scriptedRunTask: Initialize[Task[Method]] = Def.task(
-    scriptedTests.value.getClass.getMethod("run",
-                                           classOf[File],
-                                           classOf[Boolean],
-                                           classOf[Array[String]],
-                                           classOf[File],
-                                           classOf[Array[String]],
-                                           classOf[java.util.List[File]])
-  )
+  private[this] val fCls = classOf[File]
+  private[this] val bCls = classOf[Boolean]
+  private[this] val asCls = classOf[Array[String]]
+  private[this] val lfCls = classOf[java.util.List[File]]
+  private[this] val iCls = classOf[Integer]
```

Comment:
Any particular reason these aren't defined inside `scriptedRunTask`?

IIUC there's no computation done for these; they're just class literals.